api/src/test/java/org/apache/iceberg/expressions/TestMiscLiteralConversions.java

```diff
@@ -99,8 +117,10 @@ public void testInvalidBooleanConversions() {
         Types.DoubleType.get(),
         Types.DateType.get(),
         Types.TimeType.get(),
-        Types.TimestampType.withZone(),
```

Comment:
Why reorder these types? It would be fewer changes if you didn't move this.

Reply:
Reverted.